<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>861</width>
       <height>500</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border-radius:20px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_JoinLogo">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>201</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 17pt &quot;맑은 고딕&quot;;
background: transparent;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>비밀번호 찾기</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_id">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Cascadia Code SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 245, 245);
font: 75 9pt &quot;Cascadia Code SemiBold&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pw">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>230</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>전화번호</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_pw">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>260</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 245, 245);
font: 75 9pt &quot;맑은 고딕&quot;;
</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_accept">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>350</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;맑은 고딕&quot;;
background-color: rgb(255, 221, 166);
border-radius:15px;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>확인</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>-50</x>
       <y>-20</y>
       <width>1171</width>
       <height>641</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>파스텔1.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>270</y>
       <width>501</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../이미지/토성중간1.png</pixmap>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>frame_3</zorder>
    <zorder>label_4</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
